README.md

```diff
@@ -46,6 +46,14 @@ bash download_dataset.sh {GCS_PROJECT} {GCS_BUCKET_NAME}
 
 To run maxtext the TPUVMs must have permission to read the gcs bucket. These permissions are granted by service account roles, such as the `STORAGE ADMIN` role. 
 
+## Getting Started: Configure Diagnostic Settings
+Here is the related PyPI package: https://pypi.org/project/cloud-tpu-diagnostics.
+### Debugging
+The following configurations will help to debug a fault or when a program is stuck or hung somewhere by collecting stack traces. Change the parameter values accordingly in `MaxText/configs/base.yml`:
+1. Set `collect_stack_trace: True` to enable collection of stack traces on faults or when the program is hung. To disable this, set `collect_stack_trace: False`.
+2. Set `stack_trace_to_cloud: False` to display stack traces on console. `stack_trace_to_cloud: True` will create a temporary file in `/tmp/debugging` in the TPUs to store the stack traces. There is an agent running on TPU VMs that will periodically upload the traces from the temporary directory to cloud logging in the gcp project.
```

Comment:
`stack_trace_to_cloud=True` - create a temp file in `/tmp/debugging` directory and dump the traces to it. So, you can view the traces in VM itself. Additionally, runtime monitor agent will upload the traces from temp file to a dedicated log name in cloud logging. And, that log name will only have traces and no other information.
`stack_trace_to_cloud=False` - stack traces are displayed on console and moved to cloud logging but it will be present inside stdout log name along with other standard output.

Reply:
Oh thank you for this explanation this is much clearer. I see the value in having a log name with only the traces - I think you should include that in the comment for stack_trace_to_cloud - that there is a dedicated log for only the stack traces. Is it obvious to the user where to find this log?

Reply:
Good point. I will add this information to PyPI README.